src/test/java/ai/rev/speechtotext/unit/RevAiStreamingClientContentTypeTest.java

```diff
@@ -80,7 +81,7 @@ public void setup() {
     streamContentType.setChannels(channels);
     this.streamContentType = streamContentType;
 
-    streamingClient = new StreamingClient(EnvHelper.getToken());
```

Comment:
unit tests dont need a real access token, this way they dont require the environment variable to be set